CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift

```diff
@@ -103,9 +103,14 @@ public final class Camera2ViewController: UIViewController, CameraScreen {
 
     fileprivate func configureTitle() {
         if UIDevice.current.isIphone {
-            self.title = NSLocalizedStringPreferredFormat(
-                "ginicapture.navigationbar.camera.title",
-                comment: "Info label")
+            if giniConfiguration.onlyQRCodeScanningEnabled {
+                title = NSLocalizedStringPreferredFormat("ginicapture.camera.infoLabel.only.qr",
+                                                         comment: "Info label")
+            } else {
+                self.title = NSLocalizedStringPreferredFormat(
```

Comment:
[suggestion]: Maybe we can agree to use just one of the ways of writing this.
I personally prefer like this : 
``` swift 
title = NSLocalizedStringPreferredFormat("ginicapture.navigationbar.camera.title",
                                         comment: "Info label")
``` 
not  like
``` swift
self.title = NSLocalizedStringPreferredFormat(
                   "ginicapture.navigationbar.camera.title",
                    comment: "Info label")
```

Reply:
<3